cli/templates/js/api/src/functions/stripeWebhook/stripeWebhook.js

```diff
@@ -18,12 +16,9 @@ import { handleStripeWebhooks } from '@redwoodjs-stripe/api'
  * @param { Context } context - contains information about the invocation,
  * function, and execution environment.
  */
-
-/*
- * Stripe documentation recommends making any calls to db for syncing inside of webhooks
- */
```

Comment:
`event` and `context` won't be typed unless the comment above is right next to the function definition